pokemon_load_data.py

```diff
@@ -47,59 +46,51 @@ async def load_data():
-        
-        if existing_pokemon:
-            # Update existing Pokémon
-            existing_pokemon.name = entry.get("Name")
-            existing_pokemon.type_1 = entry.get("Type 1")
-            existing_pokemon.type_2 = entry.get("Type 2") if entry.get("Type 2") else None
-            existing_pokemon.total = entry.get("Total", 0)
-            existing_pokemon.hp = entry.get("HP", 0)
-            existing_pokemon.attack = entry.get("Attack", 0)
-            existing_pokemon.defense = entry.get("Defense", 0)
-            existing_pokemon.sp_atk = entry.get("Sp. Atk", 0)
-            existing_pokemon.sp_def = entry.get("Sp. Def", 0)
-            existing_pokemon.speed = entry.get("Speed", 0)
-            existing_pokemon.generation = entry.get("Generation", 0)
-            existing_pokemon.legendary = entry.get("Legendary", False)
-        else:
-            # Insert new Pokémon with the unique number
-            pokemon = Pokemon(
-                number=number,
-                name=entry.get("Name"),
-                type_1=entry.get("Type 1"),
-                type_2=entry.get("Type 2") if entry.get("Type 2") else None,
-                total=entry.get("Total", 0),
-                hp=entry.get("HP", 0),
-                attack=entry.get("Attack", 0),
-                defense=entry.get("Defense", 0),
-                sp_atk=entry.get("Sp. Atk", 0),
-                sp_def=entry.get("Sp. Def", 0),
-                speed=entry.get("Speed", 0),
-                generation=entry.get("Generation", 0),
-                legendary=entry.get("Legendary", False)
-            )
-            session.add(pokemon)
+        # Create a dictionary of Pokémon data for bulk insert
+        pokemon_data.append({
+            'number': number,
+            'name': entry.get("Name"),
+            'type_1': entry.get("Type 1"),
+            'type_2': entry.get("Type 2") if entry.get("Type 2") else None,
+            'total': entry.get("Total", 0),
+            'hp': entry.get("HP", 0),
+            'attack': entry.get("Attack", 0),
+            'defense': entry.get("Defense", 0),
+            'sp_atk': entry.get("Sp. Atk", 0),
+            'sp_def': entry.get("Sp. Def", 0),
+            'speed': entry.get("Speed", 0),
+            'generation': entry.get("Generation", 0),
+            'legendary': entry.get("Legendary", False)
+        })
         
-        # Commit changes for both new and updated records
-        try:
-            session.commit()
-        except Exception as e:
-            session.rollback()
-            print(f"Error processing Pokémon with number {number}: {e}")
+        # Add the new unique number to the set of existing numbers
+        existing_numbers.add(number)
 
-    session.close()
\ No newline at end of file
+    # Perform bulk insert using bulk_insert_mappings
+    try:
+        session.bulk_insert_mappings(Pokemon, pokemon_data)
+        session.commit()
+        print("Data inserted successfully.")
```

Comment:
return the response for this route, and mention how many records are inserted into the Database